src/components/screens/explorer/transactions/transactionDetailView/transactionDetailView.js

```diff
@@ -76,13 +77,19 @@ class TransactionDetailView extends React.Component {
         {children}
         { transaction.type === transactionTypes.send
           && (transaction.asset && transaction.asset.data) ? (
```

Comment:
- 🐛 Message and Size is not displayed if the message is empty

Reply:
Resolved